src/liger_kernel/transformers/__init__.py

```diff
@@ -15,6 +15,7 @@
     apply_liger_kernel_to_mixtral,
     apply_liger_kernel_to_phi3,
     apply_liger_kernel_to_qwen2,
+    apply_liger_kernel_to_qwen2_vl,
```

Comment:
will this break users if transformers < 4.44?

Reply:
The import won't break, but if someone tries to call the function then they would get an import error as it tries to import the Qwen2-VL module internally.